feature-libs/checkout/base/styles/components/steps/payment/_payment-method.scss

```diff
@@ -62,18 +56,7 @@
 
   .cx-checkout-text {
     @include media-breakpoint-down(md) {
-      padding-inline-start: 3.5rem;
-
-      @include forVersion(6.3) {
-        padding-inline-start: 0;
-      }
-    }
-    @include media-breakpoint-down(sm) {
```

Comment:
Did you intentionally remove this?

Reply:
Yes. With the 6.3 version having padding (padding-inline-start: 0;) in both break points we have no need to set the same value twice.